src/components/common/Calculator.tsx

```diff
@@ -49,6 +52,17 @@ const Calculator = ({ standalone = false }: Props): JSX.Element => {
   const [scientificMode, setScientificMode] = useState(false);
   const [history, setHistory] = useState<string[]>([]);
 
+  const { mutate: postAction } = mutations.usePostAppAction();
+  const { context } = useLocalContext();
+
+  const saveAction = (data: { mathjs: string }): void => {
```

Comment:
As discussed, let's call it `equation` instead of `mathjs`. And save the result!